addons/dashboard/dashboard.yaml

```diff
@@ -5,7 +5,7 @@ metadata:
   labels:
     kubeaddons.mesosphere.io/name: dashboard
   annotations:
-    catalog.kubeaddons.mesosphere.io/addon-revision: "3.0.2-0"
+    catalog.kubeaddons.mesosphere.io/addon-revision: "3.0.2-1"
     appversion.kubeaddons.mesosphere.io/dashboard: "3.0.2"
```

Comment:
feel like the app version should change....

Reply:
Agreed. The app version seems to actually be `2.2.0`.